python/doc/math_notations.sty

```diff
@@ -6,12 +6,59 @@
 
```

Comment:
I think that some motivations for the current notations will help to maintain it throughout the future versions.
```suggestion
% A random variable is upcase.
% An observation of a random variable (or a real number) is downcase.
% A random vector is in latin, upcase, bold letter.
% A vector is in latin, downcase, bold letter.
% A set is in latin, caligraphic, upcase, letter.

```